src/FunFair.CodeAnalysis.Tests/ProhibitedClassesDiagnosticsAnalyzerTests.cs

```diff
@@ -0,0 +1,159 @@
+using System.Threading.Tasks;
+using FunFair.CodeAnalysis.Tests.Helpers;
+using FunFair.CodeAnalysis.Tests.Verifiers;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.Diagnostics;
+using Xunit;
+
+namespace FunFair.CodeAnalysis.Tests
+{
+    public sealed class ProhibitedClassesDiagnosticsAnalyzerTests : CodeFixVerifier
+    {
+        protected override DiagnosticAnalyzer GetCSharpDiagnosticAnalyzer()
+        {
+            return new ProhibitedClassesDiagnosticsAnalyzer();
+        }
+
+        [Fact]
+        public Task AssertFalseWithMessageIsAllowedAsync()
```

Comment:
probaby want to give this a better name?

Reply:
Done